temporalio/contrib/openai_agents/_openai_runner.py

```diff
@@ -21,6 +22,16 @@
 from temporalio import workflow
 from temporalio.contrib.openai_agents._model_parameters import ModelActivityParameters
 from temporalio.contrib.openai_agents._temporal_model_stub import _TemporalModelStub
+from temporalio.exceptions import ApplicationError, TemporalError
+
+
+class AgentsWorkflowFailure(TemporalError):
```

Comment:
I would recommend putting this in our agent-specific `workflow` module since it has no out-of-workflow value